### DISCUSSION TITLE
Cross-sectional processing before Longitudinal

### DISCUSSION THREAD

**Discussion by dbrennan44** [ftdc-picsl/antsnetct#43] [General]
Hi there,
How much of the cross-sectional processing is necessary before the longitudinal pipeline? If I have my dataset complete, and know I'm running longitudinal processing, is there a way to disable the Cortical Thickness (even as workaround, number iterations =0 for example) step in the cross sectional to get to the longitudinal processing faster? Thanks!

**Comment by cookpa** [comment]
Hi, 

The fastest way is to set cortical thickness iterations to 1, and set the group template to "none". 

I probably should figure out a better way to do this, because it's not great having suboptimal thickness images in the cross-sectional output.
